app/javascript/styles/mastodon/components.scss

```
@@ -6624,22 +6624,26 @@ noscript {
 
     &__content {
       white-space: pre-wrap;
+      margin-top: 5px;
     }
 
     strong {
       font-size: 15px;
       font-weight: 500;
     }
 
-    .icon-button {
-      color: $primary-text-color;
+    &__buttons {
+      button:hover {
```

```suggestion
      button:hover span {
```

So that it won't underline *empty* space.